leo/plugins/qt_frame.py

```diff
@@ -1194,7 +1194,7 @@ def radio_button_callback(
         if not find.node_only and not find.suboutline_only and not find.file_only:
             w = self.radio_button_entire_outline
             w.toggle()
-    #@+node:ekr.20210923060904.1: *3* ftm.init_widgets_from_dict (new)
+    #@+node:ekr.20210923060904.1: *3* ftm.init_widgets_from_dict
```

Comment:
For consistency: Node is called 'ftm.init_widgets_from_dict' while the method is called 'set_widgets_from_dict' .